config/crd/bases/dbaas.redhat.com_mongodbatlasinventories.yaml

```diff
@@ -0,0 +1,163 @@
+
```

Comment:
can you add an example of the CR for this under config/samples

Reply:
Done